src/createDatabase.js

```
@@ -0,0 +1,52 @@
+'use strict';
+/* eslint-disable */
+
+const { Sequelize } = require('sequelize');
+
+require('dotenv').config();
+
+const { POSTGRES_HOST, POSTGRES_PORT, POSTGRES_USER, POSTGRES_PASSWORD } =
+  process.env;
+
+const createDatabase = async () => {
+  // Connect to default postgres database first
+  const sequelize = new Sequelize({
+    database: 'postgres',
+    username: POSTGRES_USER || 'postgres',
+    host: POSTGRES_HOST || 'localhost',
+    dialect: 'postgres',
+    port: Number(POSTGRES_PORT) || 5432,
+    password: POSTGRES_PASSWORD || '123',
```

Using a hardcoded fallback `POSTGRES_PASSWORD || '123'` will try to authenticate with the password `'123'` when `POSTGRES_PASSWORD` is not set. If the local Postgres instance expects no password (common in local dev) or a different password, this will cause `sequelize.authenticate()` to fail and tests/setup to stop early. Consider using `undefined` (or an empty string only if appropriate) as the default so the environment determines authentication, for example:

```javascript
password: POSTGRES_PASSWORD || undefined
```

Or enforce that the env var must be set and fail early with a clear message so tests aren't confused by silent auth errors.